core/profiles/src/main/java/software/amazon/awssdk/profiles/internal/ProfileFileRefresher.java

```diff
@@ -127,7 +130,10 @@ private boolean canReloadProfileFile() {
 
         try {
             Instant lastModifiedInstant = Files.getLastModifiedTime(profileFilePath).toInstant();
-            return currentRefreshRecord.refreshTime.isBefore(lastModifiedInstant);
+            boolean canReloadFile = currentRefreshRecord.refreshTime.isBefore(lastModifiedInstant);
+            log.info("For path {}, with previous refreshTime {}, last modified time {}, canReloadProfileFile is {}",
```

Comment:
If the build is passing, Shall I remove this?
I added it only for debugging, There are few other logs which need to removed, or log level need to be changed

Also, instead of thread sleep.. in the tests should we cleanup and directly update the lastModifiedTime?
```
Path credentialsFilePath2 = generateTestCredentialsFile(parentDirectory,"modifiedAccess", "modifiedSecret");
        Instant lastModifiedTime = Files.getLastModifiedTime(credentialsFilePath2).toInstant();
        lastModifiedTime = lastModifiedTime.plus(5, ChronoUnit.MINUTES);
        Files.setLastModifiedTime(credentialsFilePath2, FileTime.from(lastModifiedTime));
```

Reply:
I would avoid relying on platform dependant functionalities in unit tests. I'd keep the log, just at a debug level.

Reply:
@L-Applin updated the logging, please review